components/bluetooth/lib.rs

```diff
@@ -754,6 +782,10 @@ impl BluetoothManager {
             None => return drop(sender.send(Err(BluetoothError::NotFound))),
         };
         let services = primary_service.get_includes(device).unwrap_or(vec!());
+
+        // TODO: Step 7: If its argument is empty, throws a NotFoundError.
```

Comment:
I think this step here is just an early return, if the services vec is empty, the function will return still with a `NotFoundError` [at the end of the function](https://github.com/szeged/servo/pull/57/files/c86e57239515812358fdea95dbae665769c6eb42#diff-0138552b0d86c05d27c9ded05f6f2f35R804).
In other functions e.g. in `get_primary_service` there is a check here too, I just missed that step here.